sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_suggest_v1/schema.yaml

```diff
@@ -0,0 +1,37 @@
+fields:
+  - mode: NULLABLE
+    name: form_factor
+    type: STRING
+  - mode: NULLABLE
+    name: send_event
+    type: STRING
+  - mode: NULLABLE
+    name: submission_date
+    type: DATE
+  - mode: NULLABLE
+    name: begin_timestamp
+    type: TIMESTAMP
+  - mode: NULLABLE
+    name: end_timestamp
+    type: TIMESTAMP
```

Comment:
if you decide to remove these from the query you'll want to also take them out here.